docs/UserGuide.adoc

```diff
@@ -295,7 +295,7 @@ Alias example:
 
 ==== Clearing all entries : `clear`
```

Comment:
Do you think it might be more apt if we group the statistics command near the social media/birthday commands? I'm not sure what statistics you're adding, but it might be more aptly placed there if it feels more like a display "social media statistics" kind of thing. Maybe place it near my access display command? I'm not sure